Backpack-SwiftUI/BottomSheet/Classes/BPKBottomSheet.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public extension View {
+    func bpkBottomSheet<BottomSheetContent: View>(
+        isPresented: Binding<Bool> = .constant(false),
+        isClosable: Bool = true,
+        closeButtonAccessibilityLabel: String? = nil,
```

Comment:
This api makes it possible to set `isClosable` to true, while leaving the accessibility label `nil`

can you create maybe 2 method to show the bottom sheet? one that will be closable (forcing passing the accessibility label) and the other not closable, where there's no accessibility label for the close button

Reply:
Thanks Nico for your idea. I have created 2 methods for these 2 cases as you said, `bpkCloseableBottomSheet` is the closeable one with a close button and accessibility label all the time. And another one is `bpkBottomSheet` which is not closeable and without accessibility label.